app/src/components/sections/SingleTeamView/forms/MintForm.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="flex flex-col gap-4">
+    <h2>Mint {{ tokenSymbol }}</h2>
+
+    <h3>Please input the {{ address ? '' : 'address and' }}amount to mint</h3>
+    <label class="input input-bordered flex items-center gap-2 input-md mt-2 w-full">
+      <p>Address</p>
+      |
+      <input
+        type="text"
+        class="grow"
+        data-test="address-input"
+        v-model="inputAddress"
+        :disabled="Boolean(address)"
+      />
+    </label>
+    <div
+      class="pl-4 text-red-500 text-sm w-full text-left"
+      v-for="error of $v.address.$errors"
+      :key="error.$uid"
+    >
+      {{ error.$message }}
+    </div>
+    <label class="input input-bordered flex items-center gap-2 input-md mt-2 w-full">
+      <p>Amount</p>
+      |
+      <input type="number" class="grow" data-test="amount-input" v-model="amount" />
+      {{ tokenSymbol }}
+    </label>
+    <div
+      class="pl-4 text-red-500 text-sm w-full text-left"
+      v-for="error of $v.amount.$errors"
+      :key="error.$uid"
+    >
+      {{ error.$message }}
+    </div>
+
+    <div class="text-center">
+      <LoadingButton v-if="loading" class="w-44" color="primary" />
+      <button v-if="!loading" class="btn btn-primary w-44 text-center" @click="onSubmit()">
+        Mint
+      </button>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import LoadingButton from '@/components/LoadingButton.vue'
+import useVuelidate from '@vuelidate/core'
+import { helpers, numeric, required } from '@vuelidate/validators'
+import { isAddress, type Address } from 'viem'
+import { onMounted } from 'vue'
+
+const inputAddress = defineModel('description')
+const amount = defineModel('amount')
```

Comment:
You don't need to make this a model. 
it can be a simple ref, so you won't have typing issue in the bottom.

Reply:
i see okay